src/Components/CreateImageWizard/steps/ImageOutput/ReleaseSelect.tsx

```diff
@@ -158,6 +158,7 @@ const ReleaseSelect = () => {
     <FormGroup isRequired={true} label="Release">
       <Select
         isOpen={isOpen}
+        onOpenChange={(isOpen) => setIsOpen(isOpen)}
```

Comment:
Nice one, thanks! Found one bug - when clicking on "Show options for..." the select closes and needs to be reopen to select the CentOS option.